### DISCUSSION TITLE
Group in tab

### DISCUSSION THREAD

**Discussion by MembershipExcel** [iSoron/uhabits#1959] [Feature Requests]
Hi
I just saw this picture ![EXEMPLE 293719781-3cf06f91-e26e-40b1-a6a2-b548f9873f68.png](https://github.com/iSoron/uhabits/assets/48916916/d9f0fc95-ca64-483f-a5a6-1a0d3c5eb783)

From [this post](https://github.com/iSoron/uhabits/discussions/1911)

And this is EXACTLY what I want but in the "manual" order mode. 

Do we already have this ?
 I just want a categorie that can collaps

Thank you

**Comment by herrbrixter** [comment]
Duplicate of #495
